client/src/modules/user/model/actions.js

```diff
@@ -163,8 +162,26 @@ export const getAccountDetails = token =>
 
 export const checkIfDataLeft = () => getData('/auth/check-if-data-left');
 
-export const deleteAccount = (email, password) =>
-  deleteData('/auth', { email, password });
+export const sendDeleteAccountMail = () => async dispatch => {
+  try {
+    const result = await getData('/auth/send-delete-account-mail');
```

Comment:
just `/auth/delete-account`

Reply:
I can't.
 
1. This action is not deleting account but sending email with delete link
2. /delete-account API endpoint is allready in use for a controller thath is deleting account
![Zrzut ekranu 2019-10-25 o 10 35 58](https://user-images.githubusercontent.com/27632432/67556387-5db39400-f713-11e9-862b-74dea17616fd.png)